app/concepts/cinema_hall/use_case/generate_seats.rb

```diff
@@ -0,0 +1,18 @@
+
+
+module UseCase
+    module CinemaHalls 
+        class GenerateSeats < UseCase::Base::Base 
+            def call(id)
+                big_alphabet = ("A".."Z").to_a
+                @cinema_hall = UseCase::CinemaHalls::FindX.new(repository)
```

Comment:
Please use the Repository for finding records. You don't need to create a new use_case for that.